lib/lte_link_control/lte_lc.c

```diff
@@ -1122,15 +1198,38 @@ int lte_lc_edrx_get(struct lte_lc_edrx_cfg *edrx_cfg)
 		return -EFAULT;
 	}
 
-	err = parse_edrx(response, edrx_cfg);
+	err = parse_edrx(response, edrx_cfg, edrx_value, ptw_value);
 	if (err) {
 		LOG_ERR("Failed to parse eDRX parameters, error: %d", err);
 		return -EBADMSG;
 	}
 
+	lte_lc_edrx_values_store(prev_lte_mode, edrx_value, ptw_value);
+
 	return 0;
 }
 
+#if CONFIG_UNITY
+void lte_lc_edrx_on_modem_cfun(int mode, void *ctx)
+#else
+NRF_MODEM_LIB_ON_CFUN(lte_lc_edrx_cfun_hook, lte_lc_edrx_on_modem_cfun, NULL);
+
+static void lte_lc_edrx_on_modem_cfun(int mode, void *ctx)
+#endif
+{
+	ARG_UNUSED(ctx);
+
+	/* If eDRX is enabled and modem is going to power off, subscription of unsolicited eDRX
+	 * notifications must be re-newed because modem forgets that information
+	 * although it stores eDRX value and PTW for both syste modes.
```

Comment:
```suggestion
	/* If eDRX is enabled and modem is powered off, subscription of unsolicited eDRX
	 * notifications must be re-newed because modem forgets that information
	 * although it stores eDRX value and PTW for both system modes.
```